tests/backtest/test_polygon.py

```diff
@@ -226,9 +226,9 @@ def test_polygon_legacy_backtest(self):
             show_plot=False,
             show_tearsheet=False,
             save_tearsheet=False,
-            polygon_api_key=POLYGON_API_KEY,  # TODO Replace with Lumibot owned API Key
+            api_key=POLYGON_API_KEY,  # TODO Replace with Lumibot owned API Key
```

Comment:
I think this rename will break a lot of existing code. Would prefer to leave the name as is

Reply:
I put it back in with a DepricationWarning and updated the legacy test to use it to prove functionality.